subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/repositories/DefaultRepositoryContentDescriptorTest.groovy

```diff
@@ -36,6 +36,13 @@ class DefaultRepositoryContentDescriptorTest extends Specification {
         IllegalArgumentException ex = thrown()
         ex.message == "Group cannot be null"
 
+        when:
+        descriptor.includeGroupAndSubGroups(null)
+
+        then:
+        ex = thrown()
+        ex.message == "Group cannot be null"
```

Comment:
🤔 I think in this case this message could be a little misleading - there are groups and subgroups and an argument named "group" in play here.  I'd consider renaming the argument to something distinct, even something generic like "spec", so that there is immediate understanding if this exception is thrown.